tests/realtikvtest/addindextest3/functional_test.go

```diff
@@ -213,6 +213,15 @@ func TestAddIndexPresplitIndexRegions(t *testing.T) {
 	checkSplitKeys(tablecodec.TempIndexPrefix|3, 12, true)
 	tk.MustExec("drop index idx on t;")
 	tk.MustExec("set @@global.tidb_ddl_enable_fast_reorg = off;")
+
+	tk.MustExec("drop table t;")
+	tk.MustExec("set @@global.tidb_ddl_enable_fast_reorg = on;")
+	tk.MustExec("set @@global.tidb_enable_dist_task = off;")
+	tk.MustExec("create table t (a int, b int) partition by range (b)" +
+		" (partition p0 values less than (10), " +
+		"  partition p1 values less than (maxvalue));")
+	tk.MustExec("alter table t add unique index p_a (a) global pre_split_regions = (by (5), (15));")
```

Comment:
maybe use comments to specify the option to check that it works

Reply:
https://github.com/pingcap/tidb/pull/58408/files#diff-097dd78eea6150a40af20fc573492ca816b0dd01a05d0c2f4174511162c050c0R196-R198